applications/GeoMechanicsApplication/tests/lysmer_boundary_column3D/README.md

```diff
@@ -0,0 +1,48 @@
+# Lysmer absorbing boundary conditions on a column made of 3D elements
+
+These 3D tests verify the P-wave propagation through 1D column.  
+
+## Setup
+
+The tests is performed for an 1D column. The column height and width/depth are 10 $m$ and 1 $m$, respectively. A
+schematic representation can be found in the figure below:
+
+![SetUp](SetUp.svg)
+
+All Nodes have fixed displacement in the lateral directions. The column bottom is fixed in all directions. The absorbing boundary conditions are applied at the bottom and the unity absorbing factors are used there. An instant load $L$ of 10 $N/m^2$ is applied at the top of the column towards the bottom.
```

Comment:
Maybe indicate which ( X and Y or X and Z ) are the lateral directions.

Reply:
re-phrased with the column axis.